ORKESerialization runs on simulator but fails to archive

I included the ORKESerialization.m and .h files in my iOS project.  I did this but dragging the 2 files from the files downloaded from GitHub, and then doing the #import "ORKESerialization.h" in the bridging header. 

I was able to use the packages successfully, building no problem and running my app in the simulator for iOS versions 9.1 and 9.3.  However when I tried to archive the project for a generic iOS device, I get the following error:

```
Undefined symbols for architecture arm64:
"_OBJC_CLASS_$_ORKLocation", referenced from: objc-class-ref in ORKESerialization.o "_OBJC_CLASS_$_ORKConfirmTextAnswerFormat", referenced from: objc-class-ref in ORKESerialization.o ld: symbol(s) not found for architecture arm64 clang: error: linker command failed with exit code 1 (use -v to see invocation)

```

I asked a question on stackoverflow.com here: http://stackoverflow.com/questions/36266388/ios-build-succeeds-but-fails-to-archive

It was suggested that this may be a bug.

I confirm the archive is failed:
1. Open ORKTest project
2. Check both target memberships for `ORKESerialization.m`
<img width="131" alt="screen shot 2016-03-29 at 11 27 54 am" src="https://cloud.githubusercontent.com/assets/11466704/14119092/61868cbc-f5a1-11e5-9fdb-ffd053e47982.png">
3. Run Archive

Error output:

```
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_ORKLocation", referenced from:
      objc-class-ref in ORKESerialization.o
  "_OBJC_CLASS_$_ORKConfirmTextAnswerFormat", referenced from:
      objc-class-ref in ORKESerialization.o
```

--- a/ResearchKit/ActiveTasks/ORKShoulderRangeOfMotionStep.h
+++ b/ResearchKit/ActiveTasks/ORKShoulderRangeOfMotionStep.h
@@ -36,6 +36,7 @@
  The `ORKShoulderRangeOfMotionStep` class represents a step that takes a range of motion measurement
  for either a left or right shoulder.
  */
+ORK_CLASS_AVAILABLE
 @interface ORKShoulderRangeOfMotionStep : ORKRangeOfMotionStep
 
 @end

--- a/ResearchKit/Common/ORKVideoInstructionStep.h
+++ b/ResearchKit/Common/ORKVideoInstructionStep.h
@@ -41,6 +41,7 @@ NS_ASSUME_NONNULL_BEGIN
  You can use video instruction steps to present video content during a task.
  
  */
+ORK_CLASS_AVAILABLE
 @interface ORKVideoInstructionStep : ORKInstructionStep
 
 /**

